Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -2677,6 +2678,14 @@ - (void)resetFrame
     self.view.frame = frame;
 }
 
+- (void)imagePicker:(OWSImagePickerGridController *)imagePicker
+    didPickImageAttachments:(NSArray<SignalAttachment *> *)attachments
+{
+    // TODO support approving multiple attachments.
+    SignalAttachment *firstAttachment = attachments.firstObject;
+    [self tryToSendAttachmentIfApproved:firstAttachment];
```

Comment:
Are we going to use one UI/VC for "approving" media galleries (with multiple captions, etc.), and other messages (e.g. mp3s, pdfs, single image, single video, etc.)?  Or will we separate VCs?